Banking/src/main/java/com/example/Banking/model/User.java

```diff
@@ -0,0 +1,31 @@
+package com.example.Banking.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "Users")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+
+    @Id
+    private long id;
+    private String firstName;
+    private String lastName;
+    private Date dob;
+    private int age;
```

Comment:
When we have dob we don't actually need age. Otherwise you need to calculate and update age yearly.